themes/scalereal/static/css/style.css

```diff
@@ -3363,4 +3363,148 @@ lottie-player {
 
 .feedback-mobile  .owl-carousel .owl-dots {
   margin-top: -2px;
+}
+
+/* Client Success Stories Section */
+.client-success-stories-section .container .box {
+  padding: auto 98px ; 
+}
+.client-success-stories-section .box::before {

```

Comment:
Except this, remove .box from everywhere. No need to add this class. Just keep it for vertical divider